### PR TITLE
Displaying evidence income on processed application page.

### PR DESCRIPTION
**Pull request by zaparka**

### CODE REVIEW

File: app/models/views/overview/application.rb [hunk 1]
```diff
@@ -46,6 +46,11 @@ def total_monthly_income
         @application.income ? format_currency(@application.income.round) : format_threshold_income
       end
 
+      def total_monthly_income_from_evidence
+        return nil if @application.evidence_check.blank?
+        format_currency(@application.evidence_check.income.round)
```
Comment (leoapost) [inline app/models/views/overview/application.rb]: I wonder why the income amount needs to be rounded?

Comment (zaparka) [reply app/models/views/overview/application.rb]: @leoapost I guess they don't want to deal with nickles.

File: app/views/shared/processed_and_deleted/_show.html.slim [hunk 2]
```diff
@@ -7,7 +7,7 @@ header
 =build_section_with_defaults 'Personal details', @applicant
 =build_section_with_defaults 'Application details', @details
 =build_section 'Benefits', @application_view, %w[benefits]
-=build_section 'Income', @application_view, %w[number_of_children total_monthly_income]
+=build_section 'Income', @application_view, %w[number_of_children total_monthly_income total_monthly_income_from_evidence]
```
Comment (garytaylor) [inline app/views/shared/processed_and_deleted/_show.html.slim]: I thought we all hated %w's ?

Comment (zaparka) [reply app/views/shared/processed_and_deleted/_show.html.slim]: @garytaylor I guess this was not refactored. I can create a ticket for refactoring these.

**Pull request merged**